CONTRIBUTING.md

```diff
@@ -15,6 +15,10 @@ and the code must pass [coding standards](./docs/contributing/coding-standards.m
 php phing standards tests tests-acceptance
 ```
 
+Your code may not infringe the copyrights of any third party.
+If you are changing a composer's dependency in composer.json or you are changing the npm dependencies in package.json, you need to reflect this change into a list of 3rd party licenses.
```

Comment:
"a list of 3rd party licenses" => "License Overview of Included 3rd Party Libraries" + LINK

Reply:
Hell yeah done 🗡 💃